packages/catalog-process/src/services/catalogService.ts

```diff
@@ -2412,6 +2414,64 @@ export function catalogServiceBuilder(
       );
       return updatedEservice;
     },
+
+    async updateSignalHubFlag(
```

Comment:
Align with operationId in spec and with other methods that `updateEServiceSomething`

```suggestion
    async updateEServiceSignalHubFlag(
```